app/assets/stylesheets/semantic-ui/views/_statistic.scss

```diff
@@ -161,6 +159,8 @@
   opacity: 1;
   width: auto;
   margin: 0em;
+  vertical-align: top;
+  font-size: 2rem;
```

Comment:
Specifying a font-size for the icon will make it the same size for all statistic sizes. It should either be specified using a relative unit or specified for each statistic size.